tests/detect-ldap-result/test.yaml

```diff
@@ -13,3 +13,12 @@ checks:
         ldap.responses[0].operation: search_result_done
         ldap.responses[0].search_result_done.result_code: size_limit_exceeded
         alert.signature_id: 1
+checks:
+  - filter:
+      count: 1
+      match:
+        pcap_cnt: 4
+        event_type: alert
+        ldap.responses[0].operation: search_result_done
+        ldap.responses[0].search_result_done.message: "Size limit exceeded"
```

Comment:
@catenacyber this seems to be way off from the keyword and increases parity b/w rule keyword and eve output. Is this ok?

Reply:
> this seems to be way off from the keyword and decreases parity b/w rule keyword and eve output.

How so ?

`ldap.responses.error_message; content:"Size limit exceeded";` looks to match `ldap.responses[0].search_result_done.message: "Size limit exceeded"`

This ( with the suricata PR ) increases parity because the eve output has a new keyword to match on

Reply:
actually, maybe I'm using the wrong word. I meant that the rule language does not map closely to the eve output. `error_message` != `search_result_done.message`